### ISSUE TITLE
Is there a command line only version, setting, or plugin?

### ISSUE THREAD

**Issue by mysmartbus** [navit-gps/navit#1126]
Would like to use Navit to generate a text file that contains the turn by turn directions to be displayed on a unix terminal. The GUI is not loaded and no images are generated.

Basic idea is I'd give Navit a start location and an end location, then Navit generates the route and displays it on the terminal window and/or saves it to a text file.

Example comand line:
> ./navit --textonly --start (address or GPS co-ordinates) --end (address or GPS co-ordinates) --vehicle car

Break down of the command line:
> --textonly <br> Do not load the GUI.
>
> --start <br> The street address or GPS co-ordinates to start the route from
>
> --end <br> The street address or GPS co-ordinates to end the route at
>
> --vehicle <br> The type of vehicle to plan the route for.

**Comment by jkoan** [comment]
Yes this is possible by setting graphics, GUI and vehicle to the null plugins. After that you can interact with it through the dbus API: https://wiki.navit-project.org/index.php/Dbus
